.github/workflows/testing.yml

```diff
@@ -1,35 +1,51 @@
 name: Pytest MSS
 
 on: [ push, pull_request ]
+env:
+  PAT: ${{ secrets.PAT }}
 
 jobs:
   Test-MSS:
     runs-on: ubuntu-latest
-    strategy:
-      max-parallel: 5
+    
+    defaults:
+      run:
+        shell: bash
+
+    container:
+      image: yourmss/mss-test:latest
 
     steps:
     - uses: actions/checkout@v2
-
-    - name: Add conda to system path
-      run: |
-        # $CONDA is an environment variable pointing to the root of the miniconda directory, action
-        echo $CONDA/bin >> $GITHUB_PATH
-
-    - name: Setup Conda
+        
+    - name: Update dependencies if changed
       run: |
-        conda config --add channels conda-forge
-        conda config --add channels defaults
-        conda create -n mssenv python=3
+        cmp -s /meta.yaml localbuild/meta.yaml || (echo Dependencies differ \
+        && cat localbuild/meta.yaml \
+        | sed -n '/^requirements:/,/^test:/p' \
+        | sed -e "s/.*- //" \
+        | sed -e "s/menuinst.*//" \
+        | sed -e "s/.*://" > reqs.txt \
+        && conda install -n mssenv --file reqs.txt --force-reinstall)
 
-    - name: Install MSS
+    - name: Run tests
+      timeout-minutes: 25
       run: |
-        source ${CONDA}/bin/activate mssenv
-        conda install mamba
-        mamba install --only-deps mss
-        conda install --file requirements.d/development.txt
-
-    - name: Test with pytest
-      run: |
-        source ${CONDA}/bin/activate mssenv
+        source /opt/conda/bin/activate mssenv
+        cd $GITHUB_WORKSPACE
         xvfb-run pytest mslib
+    
+  Update-Images:
```

Comment:
No idea whats going on here...

Reply:
This part is triggering this action here to run:
https://github.com/Open-MSS/dockerhub/blob/stable/.github/workflows/update-images.yml

That action updates the images.